src/Modules/UI/Dnn.PersonaBar.UI/admin/personaBar/scripts/main.js

```diff
@@ -1086,23 +1086,32 @@ require(['jquery', 'knockout', 'moment', '../util', '../sf', '../config', './../
                 function showPersonaBar(callback) {
                     var $personaBar = $(".personabar");
                     var $parentBody = $(parentBody);
-                    if ($parentBody.hasClass('dnnEditState')) {
-                        $personaBar.css({ left: 0, 'display': 'block' });
-                        $parentBody.animate({ marginLeft: personaBarMenuWidth }, 1, 'linear', onShownPersonaBar);
-                        callback();
-                    } else {
-                        $personaBar.show();
+                    var updateStylePersonaBar = function () {

```

Comment:
You introduce this function variable but it's not called anywhere?

Reply:
You are right. no need anymore for this.